ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageSingle.test.ts

```diff
@@ -308,4 +314,42 @@ describe('Workload CVE Image Single page', () => {
             verifyColumnManagement({ tableSelector: 'table' });
         });
     });
+
```

Comment:
Question, not necessarily a suggestion, about pro and con for visibility to have separate test file?

Reply:
Better to keep up the momentum in the direction you chose.